crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1559,11 +1607,32 @@ where
                     non_retryable_errors: group_errors(state.non_retryable_errors),
                 }
             }
-        })
+        })?;
+
+        // Use best efforts to send the cert to remaining validators.
```

Comment:
meat

Reply:
LGTM to this part of the change.

Reply:
It does seem like this could potentially make the memory issues we were seeing with QD worse, right? Probably not a blocker for now with current operating TPS/batch size